microcloud/service/service_handler.go

```diff
@@ -182,11 +182,31 @@ func (s *ServiceHandler) Bootstrap(state *state.State) error {
 }
 
 // RunAsync runs the given hook asynchronously across all services.
-func (s *ServiceHandler) RunAsync(f func(s Service) error) error {
+func (s *ServiceHandler) RunAsync(microCloudFirst bool, f func(s Service) error) error {
```

Comment:
This arg could do with being explained in the docblock.

Reply:
I've updated the comment and renamed the function to `RunConcurrent`